pkgs/top-level/release-attrpaths-superset.nix

```diff
@@ -0,0 +1,126 @@
+# This expression will, as efficiently as possible, dump a
```

Comment:
The `releaseForRelease`/`__attrsFailEvaluation` should have a description in here.

Reply:
I want to point out that `pkg.recurseForRelease==true`, `pkg.recurseForRelease==false`, and `!(pkg?recurseForRelease)` do three different things.

I'll split this into `__attrsFailEvaluation` and `__recurseIntoDerivation`.  We can make further changes if that doesn't look right.

Reply:
See 924f8a5d4fb3d58d0d1fd2857a00e670e3036257